src/Files.Uwp/Helpers/HoverDisplay/IHoverableDisplay.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Files.Uwp.Helpers.HoverDisplay
+{
+    public  interface IHoverableDisplay
```

Comment:
We don't really have a good place for this right now, but we can easily move this when we do so we can leave this as is.